idaes/power_generation/unit_models/drum1D.py

```diff
@@ -1169,3 +1169,18 @@ def calculate_scaling_factors(self):
             sf = iscale.get_scaling_factor(
                 self.deltaP[t], default=1, warning=True)
             iscale.constraint_scaling_transform(c, sf)
+
+        for t, c in self.connection_material_balance.items():
+            sf = iscale.get_scaling_factor(
+                1e-4, default=1)
```

Comment:
This is kind of minor, but do you need a line break here to keep the lines short enough?  looks like it may fit on one line.